src/v/config/configuration.cc

```diff
@@ -1561,6 +1561,34 @@ configuration::configuration()
       "cloud_storage_segment_size_target/2",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       std::nullopt)
+  , cloud_storage_max_download_throughput_per_shard(
```

Comment:
Would it make more sense for this to be per node for consistency with the limit below?

Reply:
It's easy to chose the default value per core. The value is supposed to be high. 1GiB/s should work as a default per shard. But if it's per node the default should be very different for different instance types (4 cores vs 96 makes a huge difference). Nobody should ever touch this so I think it should be fine.